### PR TITLE
Cleaned up stale references to xpro in docs

### LINKED ISSUE #7: Fix Open edX setup documentation
#6 added edX authentication to the app, but the setup documentation (`docs/configure_open_edx.md`) still includes some XPro-specific details. We should update it to make it easy for other developers to get up & runnings

### PR DESCRIPTION
**Pull request by rhysyngsun**
#### Pre-Flight checklist
N/A

#### What are the relevant tickets?
Closes #7 

#### What's this PR do?
Removes all references to xpro in the documentation. Note: some of the openedx plugins still have "xpro" in their name, that will get addressed at a later date as/if needed.

#### How should this be manually tested?
N/A

### CODE REVIEW

File: docs/configure_open_edx.md [hunk 1]
```diff
@@ -79,9 +79,9 @@ There are two options for this:
...
   - To update to a new development version without having to actually bump the package version, simply `pip uninstall -y mitxpro-openedx-extensions`, then install again
 
-### Configure xPro as a OAuth provider for Open edX
+### Configure mitX Online as a OAuth provider for Open edX
 
-In xPro:
+In mitX Online:
 
 - go to `/admin/oauth2_provider/application/` and create a new application with these settings selected:
   - `Redirect uris`: `http://<EDX_HOSTNAME>:18000/auth/complete/mitxpro-oauth2/`
```
Comment (aliceriot) [inline docs/configure_open_edx.md:87]: should this redirect uri be changed to `http://<EDX_HOSTNAME>:18000/auth/complete/mitxonline-oauth2/` or something similar?

File: docs/configure_open_edx.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ In xPro:
     http://edx.odl.local:18000/auth/complete/mitxpro-oauth2/
     http://host.docker.internal:18000/auth/complete/mitxpro-oauth2/
```
Comment (aliceriot) [inline docs/configure_open_edx.md:91]: wondering about these two as well

File: docs/configure_open_edx.md [hunk 3]
```diff
@@ -90,7 +90,7 @@ In xPro:
...
     ```
-    - _[Linux users]_ You will need redirect uris for both the local edX host alias and for the gateway IP of the docker-compose networking setup for xPro as found via `docker network inspect mitx-online_default`
+    - _[Linux users]_ You will need redirect uris for both the local edX host alias and for the gateway IP of the docker-compose networking setup for mitX Online as found via `docker network inspect mitx-online_default`
     ```shell
     http://edx.odl.local:18000/auth/complete/mitxpro-oauth2/
     http://<GATEWAY_IP>:18000/auth/complete/mitxpro-oauth2/
```
Comment (aliceriot) [inline docs/configure_open_edx.md:96]: haha and these!

File: docs/configure_open_edx.md (hunk 1 above)
Comment (aliceriot) [reply docs/configure_open_edx.md:87]: oh nvm it looks like these are set by the `social-auth-mitxpro`? is that right?

**Review by aliceriot — COMMENTED** [review]
just one question, looks good otherwise

**Comment by rhysyngsun** [comment]
@aliceriot those are statically defined by an edx plugin so they can't be changed at the moment.

**Pull request merged**